README.md

```diff
@@ -1,6 +1,6 @@
 <a name="readme-top"></a>
 <div align="center">
-  <img src="./luis-logo.png" alt="logo" width="20%"  height="auto" />
+  <img src="./luis-logo.png" alt="logo" width="30%"  height="auto" />
   <br/>
```

Comment:
https://github.com/lerfast/math-magicians/blob/708be29b4c58c330bec6cee092cfed93c848c26e/README.md?plain=1#L1-L5
- [ ] [OPTIONAL] It could be better to if you align your items to the left, to make your README looks better 😃. To fix this, you can close the `div` after line `3`, then the logo will be aligned center, it'll look like this: 

````html
<div align="center">
  <img src="./luis-logo.png" alt="logo" width="30%"  height="auto" />
</div>

<br/>
.
.
.
````

Reply:
done!!!

![image](https://github.com/lerfast/math-magicians/assets/111823606/47669c3c-dbc6-42bf-964e-c56822209d97)